sunset/code/game/gamemodes/game_mode.dm

```diff
@@ -0,0 +1,9 @@
+/datum/game_mode
+	var/list/protected_species = list() // Species that can't be traitors
+
+/datum/game_mode/proc/removes_protected_sunset(candidates)

```

Comment:
For the record I'm about 95% sure this isn't actually going to work. You might be best off pushing the antag rolling refactor we had upstream to /tg/ and then going from there.